frontend/src/api/WorkSheetAPI.js

```diff
@@ -1,11 +1,20 @@
 import api from './createApi'
+import moment from 'moment'
 
 function putWorkSheet(id, credentials) {
   return api.put(`/worksheets/${id}`, credentials)
 }
 
-function getWorkSheets() {
-  return api.get('/worksheets')
+function getWorkSheets(startDate, endDate) {
+  const minTime = startDate
+    ? moment(startDate).format('yyyy.MM.DD') + ' 00:00:00'
+    : null
+  const maxTime = endDate
+    ? moment(endDate).format('yyyy.MM.DD') + ' 23:59:59'
+    : null
+  return api.get(`/worksheets/`, {
+    params: { minTime, maxTime },
```

Comment:
Ez tudom API-s dolog, de általában UNIX epoch timestamp sokkal célszerűbb az interface-eken, mert az időzónáktól függetlenül az abszolút idő. Itt az éjfél az a böngésző időzónája szerinti éjfél lesz, tehát magyar OS-en ez most UTC+1, tehát valójában 11 órától fog szűrni. Vagy ha 00:00:00 megy be stringként akkor meg a szerveren kell figyelni, hogy az milyen időzónában van és aszerint még tolni kell. Mindenestre: Unix timestamp UTC+0-ban az igazi, amit megjelenítésnél tolunk a user időzónájába.